holoviews/core/util/__init__.py

```diff
@@ -1123,6 +1124,31 @@ def max_extents(extents, zrange=False):
     return tuple(extents)
 
 
+def find_contiguous_subarray(a, b, *, nan_equal=True):
+    """
+    Return the start index of `a` in `b` if `a` is a contiguous subarray of `b`.
+    Return -1 if not found.
+    """
+    a = np.asarray(a)
+    b = np.asarray(b)
+
+    if len(a) == 0:
+        return 0
+    if len(a) > len(b):
+        return -1
```

Comment:
This offset the categorial axis for me.


<img width="640" height="397" alt="image" src="https://github.com/user-attachments/assets/329251bc-e12f-4b10-8901-cc9e57aa548d" />

``` python
from string import ascii_uppercase

import numpy as np

import holoviews as hv

hv.extension("bokeh")
arr = np.ones((10, 12)) * np.arange(12) * np.atleast_2d(np.arange(10)).T
arr = np.arange(120).reshape(-1, 12)
hm_data = (list(map(str, range(12))), [chr(65 + i) for i in range(10)], arr)
hm_data = (list(map(str, range(12))), [*ascii_uppercase[:10]], arr)
hm_data = (range(12), [*ascii_uppercase[:10]], arr)

p = hv.HeatMap(hm_data)
p.opts(tools=["hover"])
```

Reply:
Yes, that should have been updated to None.